packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -641,7 +643,11 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
     {#if providerConnections.length === 0}
       <NoContainerEngineEmptyScreen />
     {:else if $filtered.length === 0}
-      <ContainerEmptyScreen />
+      {#if searchTerm.length > 0}
```

Comment:
you can simplify with

```suggestion
      {#if searchTerm}
```

Reply:
Fixed.